tox.ini

```diff
@@ -9,7 +9,7 @@ description = Run the test suite against Python versions
 allowlist_externals = poetry, sh
 commands_pre = poetry install --no-root --sync
 # Export path to janus executable installed in tox environment before running pytest
-commands = sh -c 'export JANUS_PATH=$(which janus); exec "$@"' _ poetry run pytest --cov aiida_mlip --import-mode importlib
+commands = sh -c 'export JANUS_PATH=$(which janus); exec "$@"' _ poetry run pytest {posargs} --cov aiida_mlip --import-mode importlib
```

Comment:
now with your fix, probably the export will not be needed anymore, can you double check?

Reply:
Seems to, I've removed it from the CI too which hopefully also works